### PR TITLE
Fixed SonarCloud findings

### PR DESCRIPTION
**Pull request by RobWin**

### CODE REVIEW

File: resilience4j-spring/src/main/java/io/github/resilience4j/bulkhead/configure/BulkheadAspect.java [hunk 1]
```diff
@@ -59,12 +59,15 @@ public void matchAnnotatedClassOrMethod(Bulkhead Bulkhead) {
...
 
 	@Around(value = "matchAnnotatedClassOrMethod(backendMonitored)", argNames = "proceedingJoinPoint, backendMonitored")
-	public Object bulkheadAroundAdvice(ProceedingJoinPoint proceedingJoinPoint, Bulkhead backendMonitored) throws Throwable {
+	public Object bulkheadAroundAdvice(ProceedingJoinPoint proceedingJoinPoint, @Nullable Bulkhead backendMonitored) throws Throwable {
 		Method method = ((MethodSignature) proceedingJoinPoint.getSignature()).getMethod();
 		String methodName = method.getDeclaringClass().getName() + "#" + method.getName();
 		if (backendMonitored == null) {
 			backendMonitored = getBackendMonitoredAnnotation(proceedingJoinPoint);
 		}
+		if(backendMonitored == null) { //because annotations wasn't found
```
Comment (Romeh) [inline resilience4j-spring/src/main/java/io/github/resilience4j/bulkhead/configure/BulkheadAspect.java]: is that even possible to happen , to have the aspect invoked despite no annotation ?

Comment (RobWin) [reply resilience4j-spring/src/main/java/io/github/resilience4j/bulkhead/configure/BulkheadAspect.java]: No don't think so.

Comment (RobWin) [reply resilience4j-spring/src/main/java/io/github/resilience4j/bulkhead/configure/BulkheadAspect.java]: But the parameter is null... always... don't understand why.

Comment (RobWin) [reply resilience4j-spring/src/main/java/io/github/resilience4j/bulkhead/configure/BulkheadAspect.java]: That's why we have to invoke `getBackendMonitoredAnnotation`

**Pull request merged**